Classes/Runtime/FusionObjects/RuntimeFormImplementation.php

```diff
@@ -131,7 +131,8 @@ protected function renderForm(ProcessInterface $process, ActionRequest $formRequ
             null,
             null,
             'post',
-            'multipart/form-data'
+            'multipart/form-data',
+            false
```

Comment:
shouldn't this be depending on the form request?

Reply:
This is for the runtime form only. Why should this change?

Reply:
Probably I'm missing some parts here. My understanding was that the form rendered the referrer fields previously and now it is disabled with this change

Reply:
No the problem the pr adresses is hat Neos.Fusion:Form rendered Referrer Fields always which is messy for get requests. So it is made optional for forms.

The line above disables the referrer fields for the Fusion Runtime Forms (Neos.Fusion.Form:Runtime.RuntimeForm). That makes sense because there they are never used. The Runtime Form renders any error messages on its own.